src/Components/Product.js

```diff
@@ -33,6 +33,10 @@ const Product = ({query}) => {
         timestamp: Date.now(),
         tags: ['enhancement'],
       };
+      const form = new FormData();
+      form.append("features", JSON.stringify(addedFeature));
+      Service.post('/'+id+'/features', form)
+        .then(data => {});
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/elric97/feature-hunt/pullRequest?prid=8515629)
```suggestion
        .then((data) => {});
```